integration_tests/pages/induction/CheckYourAnswersPage.ts

```diff
@@ -0,0 +1,15 @@
+import Page, { PageElement } from '../page'
+import WorkAndInterestsPage from '../overview/WorkAndInterestsPage'
+
+export default class CheckYourAnswersPage extends Page {
+  constructor() {
+    super('induction-check-your-answers')
+  }
+
+  submitPage(): WorkAndInterestsPage {
+    this.submitButton().click()
+    return Page.verifyOnPage(WorkAndInterestsPage)
+  }
+
+  private submitButton = (): PageElement => cy.get('#submit-button')
```

Comment:
Not one for this PR, but since this is a common UI element it might be worth adding this to `Page` so it's available to all page types that extend it?

Reply:
Yeah, agreed.
I'm in the middle of the next PR to do with the CIAG migration (short question set to long question set), which is already quite a big change set 😬 So I'd rather not do it in my current piece of work.

I'll add a ticket to the backlog to address the submit button duplication so we don't forget it 👍

Reply:
I have created RR-770 👍